src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -411,7 +412,7 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
     completionWork onFailure {
       case e =>
         logger.error(s"Preemption work failed for call ${callKey.tag}! " + e.getMessage, e)
-        scheduleTransition(WorkflowFailed)
+        self ! CheckForWorkflowComplete
```

Comment:
I replaced everywhere which said `scheduleTransition(WorkflowFailed)` into a `CheckForComplete` message. I see your point and I've removed a few, but I'm not confident enough in avoiding all edge cases to unilaterally remove absolutely everything apart from after `case PersistenceSucceeded` / `Failed`